### PR TITLE
Make bug report template more clear and require less deletion

### PR DESCRIPTION
**Pull request by johnsaigle**
Modifies the issue template to make use of HTML comments. These will provide explainer text to the author without requiring them to delete a bunch of examples.

Also changes formatting.

The commented code will be displayed to the author but not to others viewing the issue.

This is what will be displayed to people reading the issue:

Screen Shot 2020-04-07 at 13 11 22

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,29 +7,35 @@ assignees: ''
...
+### What did you expect to happen?
+<!--A clear and concise description of what you expected to happen. -->
 
-**Browser Environment (please complete the following information):**
-- OS: [e.g. Plan9, HURD, Oberon, etc]
-- Browser [e.g. chrome, firefox] (note that only these two browsers are supported)
-- Version [e.g. 22]
+### Your Environment
+<!--[e.g. MacOS Catalina, Windows 10 -->
+- Your OS:
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
- Your OS:  
<!--e.g. MacOS Catalina, Windows 10 -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -7,29 +7,35 @@ assignees: ''
...
+<!-- We only support the most recent release of LORIS.
 
-**Additional context**
-Add any other context about the problem here.
+e.g. 22.0.0-->
+- LORIS Version: 
+<!-- e.g. Ubuntu 18.04, CentOS 7 -->
+- Linux distribution and Version:
+<!-- e.g. MySQL 5.7 -->
+- Database Version:
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]:
```suggestion
- Database Version:  
<!-- e.g. MySQL 5.7 -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -7,29 +7,35 @@ assignees: ''
...
-- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]
+### Server Environment (if known):
+<!-- We only support the most recent release of LORIS.
 
-**Additional context**
-Add any other context about the problem here.
+e.g. 22.0.0-->
+- LORIS Version: 
+<!-- e.g. Ubuntu 18.04, CentOS 7 -->
+- Linux distribution and Version:
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
- Linux distribution and Version:
<!-- e.g. Ubuntu 18.04, CentOS 7 -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -7,29 +7,35 @@ assignees: ''
...
-- LORIS Version: [e.g. 22.0.0]
-- Linux distribution and Version: [e.g. Ubuntu 16.04, CentOS 7]
-- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]
+### Server Environment (if known):
+<!-- We only support the most recent release of LORIS.
 
-**Additional context**
-Add any other context about the problem here.
+e.g. 22.0.0-->
+- LORIS Version:
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]: More sustainable w/o version number: 
```suggestion
- LORIS Version: 
<!-- Please note we only support the most recent major/minor release of LORIS-->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -7,29 +7,35 @@ assignees: ''
...
+<!-- Only Chrome and Firefox are supported. -->
+- Browser: 
 
-**Server Environment (if known):**
-_Note: We only support the most recent release of LORIS._
-- LORIS Version: [e.g. 22.0.0]
-- Linux distribution and Version: [e.g. Ubuntu 16.04, CentOS 7]
-- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]
+### Server Environment (if known):
+<!-- We only support the most recent release of LORIS.
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: delete this line and the next (somehow multi-line suggest wouldn't work here, sorry)

**Comment by johnsaigle** [comment]
@christinerogers These suggestions would make it inconsistent. Right now all the comments are above the headings. You've suggested moving some and not others.

As far as the version comment I think the latest release is more clear than `major/minor`. I think it's unclear to external contributors what a `major` or `minor` is. Also I think we're only doing releases now... with occasional emergency bugfixes. There is no major or minor branch anymore.

the 'e.g.' for the LORIS version is intentional -- just an example. It doesn't need updating for each release. Same with the versions for the OSes, database, etc.

**Comment by johnsaigle** [comment]
@christinerogers @driusan Would you mind taking another look at this so we can get it wrapped up for tomorrow?

**Comment by johnsaigle** [comment]
Thanks @christinerogers. I incorporated your suggestions.

@driusan Please review when you have a minute

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -7,29 +7,37 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is
+<!-- Before opening a new issue, please search for related issues or duplicates. -->
+<!-- For install questions and support for ongoing project, please contact our community listserv: loris-dev [at] bic.mni.mcgill.ca --> 
+### Describe the bug
+<!-- A clear and concise description of what the bug is -->
```
Comment (christinerogers) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
<!-- A clear and concise description of what the bug is, including links to related issues. -->
```

Comment (johnsaigle) [reply .github/ISSUE_TEMPLATE/bug_report.md]: applied

**Comment by christinerogers** [comment]
@driusan --  ready to review on master
Would be much easier to resolved/merge this from master than rebase it onto main, if you have a moment to review

**Comment by driusan** [comment]
This is a step backwards. People should not be submitting HTML comments in their bug report. It doesn't make it require less deletion, it just makes it less likely for them to do it and more likely for data to be lost if they accidentally type inside one of the HTML comment blocks.

**Pull request closed without merging**